app/src/main/res/values/strings.xml

```diff
@@ -668,6 +668,14 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_bicycle_rental_type_human">Human-operated bike rental</string>
     <string name="quest_bicycle_rental_type_shop_with_rental">Bike store that also rents bikes</string>
 
+    <string name="quest_bicycle_shop_repair_title">Can you repair bicycles here?</string>
+
+    <string name="quest_bicycle_shop_second_hand_title">What type of bicycles can be bought here?</string>
+    <string name="quest_bicycle_shop_second_hand_only_new">Only new bicycles</string>
+    <string name="quest_bicycle_shop_second_hand_new_and_used">New bicycles and second-hand bicycles</string>
+    <string name="quest_bicycle_shop_second_hand_only_used">Only second-hand bicycles</string>
+    <string name="quest_bicycle_shop_second_hand_no_bicycles">No bicycles sold in this bicycle shop</string>
```

Comment:
```suggestion
    <string name="quest_bicycle_shop_second_hand_title">Are second-hand bicycles sold here?</string>
    <string name="quest_bicycle_shop_second_hand_only_new">No, only new ones</string>
    <string name="quest_bicycle_shop_second_hand_new_and_used">Yes, alongside new ones</string>
    <string name="quest_bicycle_shop_second_hand_only_used">Yes, exclusively</string>
    <string name="quest_bicycle_shop_second_hand_no_bicycles">No bicycles sold at all</string>
```

Reply:
I had in one of drafts and I was wondering is it a bit too tricky. I guess I will go with it.

I just want to defend `No bicycles sold in this bicycle shop` vs `No bicycles sold at all` - if bicycle shop is replaced by say supermarket of greengrocer, them `No bicycles sold at all` is still applicable - but user should leave note instead.

Reply:
Shouldn't the feature name "Bicycle shop" below the quest title suffice to raise concerns if there is e.g. a supermarket there now?

But I guess that being explicit doesn't hurt. It just sounds a bit repetitive.

Reply:
> Shouldn't the feature name "Bicycle shop" below the quest title suffice to raise concerns if there is e.g. a supermarket there now?

It should but I estimate that at least 10%, maybe up to 60% of people will miss it.